packer/scylla_install_image

```diff
@@ -53,6 +53,8 @@ if __name__ == '__main__':
                         help='repository for install, specify .repo/.list file URL')
     parser.add_argument('--repo-for-update',
                         help='repository for update, specify .repo/.list file URL')
+    parser.add_argument('--scylla-manager-repo', default='http://downloads.scylladb.com/deb/ubuntu/scylladb-manager-3.0.list',
```

Comment:
http sounds bad. But then we can download from https:// ... and it's again http content! : 
```
deb  [arch=amd64,arm64] http://downloads.scylladb.com/downloads/scylla-manager/deb/ubuntu/scylladb-manager-3.0 stable main
```

ARGH!

Reply:
@mykaul We do have an open issue for that https://github.com/scylladb/scylla-pkg/issues/1479

Reply:
Updated to `https` based on https://github.com/scylladb/scylla-pkg/pull/3241 that need to get in before